connectors/uplynk/README.md

```diff
@@ -0,0 +1,69 @@
+# THEOplayer Uplynk Connector for Android
+
+The Uplynk connector provides a Uplynk integration for THEOplayer.
+
+## Installation
+
+1. Add the THEOplayer Maven repository to your project-level `settings.gradle` file:
+   ```groovy
+   dependencyResolutionManagement {
+       repositories {
+           google()
+           mavenCentral()
+           maven { url = uri("https://maven.theoplayer.com/releases/") }
+       }
+   }
+   ```
+2. Add THEOplayer, the Uplynk Ad Management SDK and the Uplynk Connector as dependencies in your module-level `build.gradle` file:
```

Comment:
There is no such thing as the "Uplynk Ad Management SDK". That's only for Yospace. 😛
```suggestion
2. Add THEOplayer and the Uplynk Connector as dependencies in your module-level `build.gradle` file:
```

Reply:
Thanks! Copy paste that i forgot to clean up)